### DISCUSSION TITLE
GitHub Education email authentication problem

### DISCUSSION THREAD

**Discussion by BrockMiller9** [community/community#50712] [GitHub Education]
Upon registering for Github Education - I am met with this prompt 
"We require applicants of Springboard Education to use one of these school-issued email addresses to apply:
- springboard.com
Please select a different email address."

However, my institution does not provide email address and I have already verfied that they are supported through the Github Education program. My account email is also my verified school email. Any suggestions?

**Comment by Florin4226** [comment]
So, do you resolved this?
